src/DataValues/QuantityValue.php

```diff
@@ -344,14 +361,18 @@ public function getOrderOfUncertainty() {
 	 *
 	 * @since 0.1
 	 *
-	 * @return int
+	 * @return int|null
 	 */
 	public function getSignificantFigures() {
 		$math = new DecimalMath();
 
 		// $orderOfUncertainty is +/- 200 -> 2; +/- 0.02 -> -2
 		$orderOfUncertainty = $this->getOrderOfUncertainty();
 
+		if ( $orderOfUncertainty === null ) {
+			return null;
```

Comment:
If no uncertainty is given, all figures should be considered significant. There is no need to return null here.